js/summary.js

```diff
@@ -72,20 +72,15 @@ H5P.Summary = (function ($, Question, JoubelUI) {
       intro: "Choose the correct statement.",
       solvedLabel: "Solved:",
       scoreLabel: "Wrong answers:",
-      behaviour: {
-        enableRetry: true,
-      },
+      enableRetry: true,
```

Comment:
You have moved the enableRetry variable out of the behaviour group, which means that it no longer satisfies the contract to parents: https://h5p.org/documentation/developers/contracts. The solution could be to create a group for behaviour in the semantics, or make sure that you override enableRetry if it received a behaviour.enableRetry from parent.